files/fr/web/api/blobbuilder/index.md

```diff
@@ -1,138 +1,100 @@
+* <code>void <a href="/fr/docs/Web/API/BlobBuilder#append()">append</a>(in String data, [optional] in String endings);</code>
+* <code>Blob <a href="/fr/docs/Web/API/BlobBuilder#getblob()">getBlob</a>([optional] in DOMString contentType);</code>
+* <code>File <a href="/fr/docs/Web/API/BlobBuilder#getfile()">getFile</a>(in DOMString name, [optional] in DOMString contentType);</code>
+
 
-### append()
+## Méthodes
 
-Ajoute le contenu de l'objet JavaScript spécifié au {{domxref("Blob")}} en construction. Si la valeur spécifiée n'est pas un {{domxref("Blob")}}, un [`ArrayBuffer`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer), ou une [`String`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/String), la valeur est forcée (coerced) à string avant d'être ajoutée au blob.
+### `append()`
 
-    void append(
-      in ArrayBuffer data
-    );
+Ajoute le contenu de l'objet JavaScript indiqué à l'objet [`Blob`](/fr/docs/Web/API/Blob) en cours de construction. Si la valeur fournie n'est pas un [`Blob`](/fr/docs/Web/API/Blob), [`ArrayBuffer`](/fr/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer), ou une chaîne de caractères [`String`](/fr/docs/Web/JavaScript/Reference/Global_Objects/String), la valeur est convertie en une chaîne de caractères avant d'être ajoutée au blob.
 
-    void append(
-      in Blob data
-    );
+```js
+void append(
+  in ArrayBuffer data
+);
 
+void append(
+  in Blob data
+);
 
-    void append(
-      in String data,
-      [optional] in String endings
-    );
+void append(
+  in String data,
+  [optional] in String endings
+);
+```
 
 #### Paramètres
 
 - `data`
-  - : Les données à ajouter au {{domxref("Blob")}} en construction.
+  - : Les données à ajouter à l'objet [`Blob`](/fr/docs/Web/API/Blob) en cours de construction.
 - `endings`
-  - : Spécifie comment les chaînes (strings) contenant des `\n` doivent être rendues. Cela peut être `"transparent"` (endings inchangés) ou `"native"` (les endings sont changés pour respecter la convention du système de fichier de l'OS de l'hôte). Par défault, la valeur est `"transparent"`.
+  - : Indique la façon dont les chaînes de caractères contenant `\n` doivent être écrites. La valeur de cet argument peut être `"transparent"` (les fins de lignes ne sont pas modifiées) ou `"native"` (les fins de lignes sont modifiées afin de correspondre à la convention utilisée par le système de fichier du système d'exploitation). La valeur par défaut est `"transparent"`.
 
-### getBlob()
+### `getBlob()`
 
-Retourne l'objet {{domxref("Blob")}} qui a été construit en utilisant les données passées en argument des différents appels à {{manch("append")}}.
+Renvoie l'objet [`Blob`](/fr/docs/Web/API/Blob) qui a été construit avec les données ajoutées lors des appels successifs à [`append()`](#append).
 
-    Blob getBlob(
-      in DOMString contentType {{optional_inline}}
-    );
+```js
+Blob getBlob(
+  in DOMString contentType {{optional_inline}}
```

Comment:
avec la macro `{{optional_inline}}` on voit "`in DOMString contentType Facultatif`" que je trouvais un peu bizarre

```suggestion
  in DOMString contentType // facultatif
```